client/src/cmdusart.c

```diff
@@ -246,22 +252,28 @@ static int CmdUsartBtFactory(const char *Cmd) {
     size_t len = 0;
     memset(data, 0, sizeof(data));
 
-    string = "AT+NAMEPM3_RDV4.0";
+    // here sets name. HC series need have equal.
+    string = "AT+NAME=PM3_RDV4.0";
```

Comment:
I wonder, 
yours has a  `=`  in `AT+NAME=PM3_`
and the old one doesn't.
`AT+NAMEPM3_*` 

That feels like a breaking change.